doc/manual/manual.tex

```diff
@@ -420,14 +491,194 @@ \subsection{Minimal Action Set}
 game. Of course, algorithm designers are encouraged to devise algorithms that don't depend 
 on this minimal action set for success.
```

Comment:
I don't think we should necessarily discourage a particular action set - we should instead encourage that people report exactly what was used for learning.